src/adapter/src/catalog/builtin.rs

```diff
@@ -2101,9 +2105,10 @@ AS SELECT
 FROM mz_catalog.mz_roles r",
 };
 
-pub const MZ_SYSTEM: BuiltinRole = BuiltinRole {
-    name: "mz_system",
-    id: 0,
+pub const MZ_SYSTEM: BuiltinRole = BuiltinRole { name: "mz_system" };
+
+pub const MZ_HTTP_DEFAULT_USER: BuiltinRole = BuiltinRole {
```

Comment:
Is this unused? Should it be added in a later PR?

Reply:
I thought it might be good to test a second builtin role, but you're probably right. I should save this for a later PR.